agents/locobot/saved_trajectories.py

```diff
@@ -0,0 +1,116 @@
+import os
+import copy
+import time
+import shutil
+import json
+import math
+import cv2
+import numpy as np
+import open3d as o3d
+from droidlet import dashboard
+from droidlet.dashboard.o3dviz import o3dviz
+from droidlet.lowlevel.hello_robot.hello_robot_mover import HelloRobotMover
+
+if __name__ == "__main__":
```

Comment:
what does this file do? is it referenced anywhere else in PR?  am assuming this is one-off for yourself that doesn't need review?

Reply:
it's a file that showcases how to take recorded offline trajectores, and register/visualize them.
It doesn't need to be reviewed right now.